picard/config.py

```diff
@@ -65,13 +65,6 @@ def remove(self, key):
     def raw_value(self, key):
         """Return an option value without any type conversion."""
         value = self.__config.value("%s/%s" % (self.__name, key))
-
-        # XXX QPyNullVariant does not exist in all PyQt versions, and was
-        # removed entirely in PyQt5. See:
-        # http://pyqt.sourceforge.net/Docs/PyQt5/pyqt_qvariant.html
-        if str(type(value)) == "<class 'PyQt4.QtCore.QPyNullVariant'>":
```

Comment:
http://pyqt.sourceforge.net/Docs/PyQt5/pyqt_qvariant.html

According to this it will just convert it to a python object. So I suppose it will just return a str type?